test/parser.test.js

```diff
@@ -1055,7 +1085,7 @@ describe('parser', () => {
             return content.split(';').map(key => ({ key }))
           }
 
-          on() {}
+          on() { }
```

Comment:
Already disabled prettifier in vscode, somehow this line still get's auto formatted. Let me know if I should spend time fixing both lines.

Reply:
Well if you feel like making a PR to actually add Prettifier to the project, I'm all for it!

Reply:
Yeah ESlint + Prettier would be cool. If I have the time, I'll PR it :)